arcgis-ios-sdk-samples/Maps/Download preplanned map/DownloadPreplannedMap.storyboard

```diff
@@ -0,0 +1,157 @@
+                                <items>
+                                    <barButtonItem title="Item" style="plain" id="MRq-RD-4Yt">
+                                        <activityIndicatorView key="customView" hidden="YES" opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" hidesWhenStopped="YES" style="white" translatesAutoresizingMaskIntoConstraints="NO" id="9e3-Rl-Roi">
+                                            <rect key="frame" x="20" y="17" width="20" height="20"/>
+                                            <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        </activityIndicatorView>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="ONj-3x-aZn"/>
+                                    <barButtonItem enabled="NO" title="Select Map" id="T6C-dm-3bb">
+                                        <connections>
+                                            <segue destination="Mtd-Rb-acK" kind="popoverPresentation" popoverAnchorBarButtonItem="T6C-dm-3bb" id="fVA-cM-WRO">
+                                                <popoverArrowDirection key="popoverArrowDirection" up="YES" down="YES" left="YES" right="YES"/>
+                                            </segue>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="ITk-QV-Mwr"/>
+                                    <barButtonItem enabled="NO" systemItem="trash" id="acg-XN-HC2">
+                                        <connections>
+                                            <action selector="removeDownloadsTapped:" destination="eym-3Z-3ck" id="pdB-1P-gPh"/>
+                                        </connections>
+                                    </barButtonItem>
+                                </items>
+                            </toolbar>
+                        </subviews>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="pfV-1x-ZGz" firstAttribute="bottom" secondItem="nE8-T5-4kh" secondAttribute="bottom" id="71E-cF-jot"/>
+                            <constraint firstAttribute="top" secondItem="pfV-1x-ZGz" secondAttribute="top" id="BTy-bL-9G1"/>
+                            <constraint firstItem="pfV-1x-ZGz" firstAttribute="width" secondItem="nE8-T5-4kh" secondAttribute="width" id="NCx-To-iaT"/>
```

Comment:
These `width` and `height` constraints are not necessary as the `leading`/`trailing` and `top`/`bottom` constraints specify a width and height for the map view. An equivalent alternative configuration would be to specify `centerX`/`width` and `centerY`/`height` constraints.

Reply:
Agreed : [954f538b](https://github.com/Esri/arcgis-runtime-samples-ios/pull/758/commits/954f538bba4c81adb29a06e436feee66eaad5338).